bigtable/tests/system.py

```diff
@@ -378,7 +380,7 @@ def test_read_large_cell_limit(self):
         row = self._table.row(ROW_KEY)
         self.rows_to_delete.append(row)
 
-        number_of_bytes = 10 * 1024 * 1024
+        number_of_bytes = 10 * 1024
         data = b'1' * number_of_bytes  # 10MB of 1's.
```

Comment:
If we really mean to drop the data size from 10MB to 1 MB, then we need also to update the comment on the next line accordingly.